docs/source/conf.py

```diff
@@ -0,0 +1,121 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+import setuptools_scm
+
+# Used when building API docs, put the dependencies of any
+# class you are documenting here
+autodoc_mock_imports = []
+
+# Add the module path to sys.path here.
+# If the directory is relative to the documentation root,
+# use os.path.abspath to make it absolute, like shown here.
+sys.path.insert(0, os.path.abspath("../.."))
+
+project = "wazp"
+copyright = "2022, UCL"
+author = "UCL"
```

Comment:
```suggestion
copyright = "2022, The Sainsbury Wellcome Centre"
author = "The Sainsbury Wellcome Centre and The Centre for Advanced Research Computing, UCL"
```
[The license is the SWC](https://github.com/SainsburyWellcomeCentre/WAZP/blob/main/LICENSE#L3). Not that it matters if SWC is a member of UCL but I'd keep it consistent.

And maybe "Author" should be both? What do you think?

Reply:
I'm fine with this, but @adamltyson may know more

Reply:
I don't think what we write here has any effect upon IP law, but AFAIK UCL (as the legal entity) owns copyright, so everything should probably reflect that.

Reply:
I think copyright - UCL, and author - SWC & ARC is nice.

Reply:
Should I also change the copyright to UCL in BSD-3?

Reply:
Yup

Reply:
Done